homeassistant/components/evohome/__init__.py

```diff
@@ -614,13 +614,12 @@ def __init__(self, evo_broker, evo_device) -> None:
     @property
     def current_temperature(self) -> Optional[float]:
         """Return the current temperature of a Zone."""
-        if not self._evo_device.temperatureStatus["isAvailable"]:
-            return None
-
         if self._evo_broker.temps:
-            return self._evo_broker.temps[self._evo_device.zoneId]
+            if self._evo_broker.temps[self._evo_device.zoneId] != 128:
```

Comment:
Combine those?

```py
if self._evo_broker.temps and self._evo_broker.temps[self._evo_device.zoneId] != 128:
```

Reply:
Yeah - stupid mistake (and it got through the linters?) - I will submit another PR.

Reply:
see PR #38294